prelude-nri/elm/BUCK

```diff
@@ -26,3 +26,33 @@ genrule(
     executable = True,
     visibility = ["PUBLIC"],
 )
+
+ELM_FORMAT_URL = select({
```

Comment:
We can't use nix for this?

Reply:
I’m not sure how we’re going to use Nix for this. I’d like to eventually but it’s a bigger job to figure out how it works with Buck’s download cache. For now, it seems fine to download the right binaries.